.cicd/scripts/srw_test.sh

```diff
@@ -17,5 +17,120 @@ else
     workspace="$(cd -- "${script_dir}/../.." && pwd)"
 fi
 
+we2e_experiment_base_dir="${workspace}/experiments"
```

Comment:
@jessemcfarland The script `get_expts_status.sh` in `ufs-srweather-app/regional_workflow/tests/WE2E` does things similar to this script (but not exactly the same).  You can get help for that by using `get_expts_status.sh --help`.  It would be good to combine the two so the automated tests and any manual tests the user runs use the same set of scripts.

Reply:
@gsketefian thanks! I'll check it out and try to incorporate it here.

Reply:
There is also `setup_WE2E_tests.sh` under `regional_workflow/tests/WE2E` that is used to run 10 regression tests in `machine_suites` on hera/jet using ci-hera/jet-* labels. I think modifying that for other platforms should avoid duplicate work.

Reply:
There is also setup_WE2E_tests.sh under regional_workflow/tests/WE2E that is used to run 10 regression tests in machine_suites on hera/jet using ci-hera/jet-* labels. I think modifying that for other platforms should avoid duplicate work.